src/ZODB/tests/blob_transaction.txt

```diff
@@ -402,14 +398,7 @@ Now, the serial for the existing blob should be the same:
     >>> blob_storage.load(blob._p_oid, '') == (olddata, oldserial)
     True
 
-And we shouldn't be able to read the data that we saved:
-
-    >>> blob_storage.loadBlob(blob._p_oid, s1)
-    Traceback (most recent call last):
-    ...
-    POSKeyError: 'No blob file at <BLOB STORAGE PATH>'
-
-Of course the old data should be unaffected:
```

Comment:
I could keep this in the handle_serial branch: https://github.com/zopefoundation/ZODB/commit/4558d9398b69c54950c233dc981607939956e111#diff-f3c7ea79af896e7fe5d1dfea68a004fcR403